x-pack/plugin/sql/qa/single-node/src/test/java/org/elasticsearch/xpack/sql/qa/single_node/JdbcDocCsvSpecIT.java

```diff
@@ -78,6 +79,7 @@ protected boolean logEsResultSet() {
 
     @Override
     protected final void doTest() throws Throwable {
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
I don't think that's necessary, did it cause any problems?

Reply:
In `csvjdbc`, it parse time string with `SimpleDateFormat`. In the constructor of `SimpleDateFormat`, calendar will be initialized by  the `initializeCalendar` function.
```java
private void initializeCalendar(Locale loc) {
        if (calendar == null) {
            assert loc != null;
            // The format object must be constructed using the symbols for this zone.
            // However, the calendar should use the current default TimeZone.
            // If this is not contained in the locale zone strings, then the zone
            // will be formatted using generic GMT+/-H:MM nomenclature.
            calendar = Calendar.getInstance(TimeZone.getDefault(), loc);
        }
    }

```
And In `parseTime` function, it will get milliseconds from `parsedDate`. In different TimeZone, there will be different milliseconds.
```java
public Time parseTime(String str){              
	java.util.Date parsedDate = simpleTimeFormat.parse(str);
	long millis = parsedDate.getTime();
	sqlResult = new Time(millis);
	return sqlResult;
}
```
Is there a better way to solve this problem?

Reply:
Cant' you use the timezone parsing pattern here: https://github.com/elastic/elasticsearch/pull/55223/files#diff-3dd1dabda293f7c99cda3ed7e09b2cacR66
```
csvProperties.setProperty("timeFormat", "HH:mm:ss.SSSX");
```

Reply:
It can't work.

Reply:
I tried with your branch and it worked.

Reply:
Sorry, it works, I made an another mistake.